staging/src/k8s.io/apiserver/pkg/server/hooks.go

```diff
@@ -56,6 +56,11 @@ type PostStartHookProvider interface {
 	PostStartHook() (string, PostStartHookFunc, error)
 }
 
+type PostStartHook struct {
```

Comment:
not used ?

Reply:
yep, leftover from previous attempts, thx